worker-core/src/main/java/com/github/workerframework/core/WorkerApplication.java

```diff
@@ -78,6 +78,8 @@ public final class WorkerApplication extends Application<WorkerConfiguration>
 {
     private final long startTime = System.currentTimeMillis();
     private static final Logger LOG = LoggerFactory.getLogger(WorkerApplication.class);
+    private static final long SHUTDOWN_DURATION = 5 * 60 * 1000; // 5 minutes in milliseconds
+    public static final int SHUTDOWN_LOG_INTERVAL = 15_000; // 15 seconds in milliseconds
```

Comment:
I assume this is not meant to be `public`?

```suggestion
    private static final int SHUTDOWN_LOG_INTERVAL = 15_000; // 15 seconds in milliseconds
```